docs/quickstart/assessment/standalone_assessment.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2021 Learnosity, Apache 2.0 License
+# SPDX-License-Identifier: Apache-2.0
+#
+# Basic example of embedding a standalone assessment using Items API
+# with `rendering_type: "assess"`.
+
+# Include server side Learnosity SDK, and set up variables related to user access
+from learnosity_sdk.request import Init
+from learnosity_sdk.utils import Uuid
+from .. import config # Load consumer key and secret from config.py
+# Include web server, time, and Jinja templating libraries.
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import time
```

Comment:
Is `time` still needed? I don't see it used anywhere in the code. If not, we should remove it from here and the README.md.

Reply:
Seems to be superfluous. Good one. Removed.